e2e/test_module.py

```diff
@@ -62,20 +66,27 @@ def prepare_tests():
 tests = prepare_tests()
 
 
-@pytest.mark.parametrize("test_dir", tests)
-def test_end2end(test_dir, db):
-    db.drop_database()
+def _load_yaml(path: Path) -> Dict:
+    """
+    Load YAML based file in Python dictionary.
+    """
+    file_handle = path.open("r")
+    return yaml.load(file_handle, Loader=yaml.Loader)
 
-    input_cyphers = test_dir.joinpath(TestConstants.INPUT_FILE).open("r").readlines()
+
+def _execute_cyphers(input_cyphers: List[str], db: Memgraph):
+    """
+    Execute commands against Memgraph
+    """
     for query in input_cyphers:
         db.execute_query(query)
```

Comment:
This part is wrong, it should be:
 `db.execute(query)`

Reply:
It's a comment for the other PR. Please update, because this one is not using `gqlalchemy`

Reply:
Okay 👍🏻